specification/containerservice/resource-manager/Microsoft.ContainerService/fleet/preview/2023-03-15-preview/fleets.json

```diff
@@ -1691,7 +1691,7 @@
       "properties": {
         "type": {
           "$ref": "#/definitions/ManagedClusterUpgradeType",
-          "description": "The upgrade type.\nFull requires the KubernetesVersion property to be set.\nNodeImageOnly requires the KubernetesVersion property not to be set."
+          "description": "The upgrade type.\nFull requires the KubernetesVersion property to be set.\nNodeImageOnly requires the KubernetesVersion property not to be set.\nControlPlaneOnly requires the AgentPool KubernetesVersion property not to be set."
```

Comment:
what does this mean? we don't have an AgentPool KubernetesVersion.
ControlPlaneOnly requires the KubernetesVersion to be set on our spec.